.env

```diff
@@ -30,6 +30,7 @@ MON_LOG_TRANSFORMER_VERSION=latest
 MON_ELASTICSEARCH_INIT_VERSION=latest
 MON_LOG_API_VERSION=master-20170821-224127
 MON_KIBANA_VERSION=4.6.3-master
+MON_LOGSPOUT_VERSION=latest
```

Comment:
These should be explicit versions instead of just ```latest```

Reply:
Done